pydocs/source/index.rst

```diff
@@ -9,7 +9,12 @@ XmsInterp |version|
 
 **Last Updated:** December 5, 2018
```

Comment:
Do we need to increment this?

Reply:
we could, but lets just do it on master after it is merged.